Wobble/Graphics/UI/Form/Textbox.cs

```diff
@@ -548,8 +860,10 @@ private void HandleEnter()
 
                 // Clear text box.
                 RawText = "";
+                CursorPosition = 0;
                 Selected = false;
                 ReadjustTextbox();
+                UpdateSelectedSprite();
```

Comment:
You're calling these three in sequence a lot. You could put them into its own method.